openrouteservice/src/test/resources/ors-config.json

```diff
@@ -192,6 +192,15 @@
                 },
                 "RoadAccessRestrictions": {
                   "use_for_warnings": true
+                },
+                "HereTraffic": {
+                  "enabled": false,
+                  "streets": "src/main/files/here_traffic/germany_traffic/Streets.shp",
```

Comment:
I believe files which are not being included in the repo shouldn't be referenced in the config, unless I'm missing sth here?

Reply:
This is one of the missing mock files that should be added.